misspell.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+if [ "$(misspell -locale US . | grep -v 'vendor/')" ]; then
```

Comment:
No, because we don't care that misspell returns an error code, we are looking at the actual output.